scripts/pkg/scripts/systemd-entrypoint

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# This wrapper script allows SystemD to feed a file containing a passphrase into
+# the main OpenSearch startup script
+
+if [ -n "$OPENSEARCH_KEYSTORE_PASSPHRASE_FILE" ] ; then
```

Comment:
Why is this a conditional? Aren't we building something predictable where there's always such a file?